tests/flow/more_react/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -126,7 +126,7 @@ var App = require(\"App.react\");
 
 var app = (
   <App y={42}>
-     // error, y: number but foo expects string in App.react
+    {\" \"}// error, y: number but foo expects string in App.react
```

Comment:
this highlights that this "comment" was never actually a comment. not problematic, just confusing.